src/js/form-builder.js

```diff
@@ -686,7 +686,7 @@ function FormBuilder(opts, element, $) {
         } else if (attrData.hasOwnProperty('value') || attrData.hasOwnProperty('checked')) {
           isChecked = attrData.value || attrData.checked || false
         }
-        return boolAttribute(attr, { ...attrData, [attr]: isChecked }, { first: attrData.label })
```

Comment:
doesn't this override the `attrData.label` configuration?

Reply:
I don't believe so. We calculate the label value on line 704 using either the translated value of falling back onto raw .label value.
https://github.com/kevinchappell/formBuilder/blob/27abe220d4dabe8d08a2ab64c28d8e98a986854f/src/js/form-builder.js#L704

This change mirrors how string and select attributes calculate the label's value

String and Select attributes use the i18n value directly:
https://github.com/kevinchappell/formBuilder/blob/27abe220d4dabe8d08a2ab64c28d8e98a986854f/src/js/form-builder.js#L740

https://github.com/kevinchappell/formBuilder/blob/27abe220d4dabe8d08a2ab64c28d8e98a986854f/src/js/form-builder.js#L794

Number attribute does it slightly different:
https://github.com/kevinchappell/formBuilder/blob/27abe220d4dabe8d08a2ab64c28d8e98a986854f/src/js/form-builder.js#L888